libraries/Microsoft.Bot.Streaming/Payloads/Disassemblers/PayloadDisassembler.cs

```diff
@@ -44,7 +44,9 @@ public PayloadDisassembler(IPayloadSender sender, Guid id)
 
         public abstract Task<StreamWrapper> GetStreamAsync();
 
+#pragma warning disable CA1801 // Review unused parameters
         public async Task DisassembleAsync(CancellationToken cancellationToken = default(CancellationToken))
+#pragma warning restore CA1801 // Review unused parameters
         {
             var w = await GetStreamAsync().ConfigureAwait(false);
```

Comment:
>GetStreamAsync [](start = 26, length = 14)

should be passing cancelation token to this.

Reply:
The abstract method doesn't have cancellationToken, let me see if adding it passes the binary compat check

Reply:
Can't add cancellationToken to GetStreamingAsync without breaking binary compat :(